README.md

```diff
@@ -107,6 +107,42 @@ hatch run test
 hatch run test tests/test_<SOME_FILE>.py
 ```
 
+### Integration Tests
+
+```sh
+# Run all tests
+hatch run integration-test
+
+# Run a single test file
+hatch run integration-test integration-test/test_<SOME_FILE>.py
+```
+
+To run integration tests on your local machine, you need to set up your Kaggle API credentials. There are two ways to do this:
+
+Using Environment Variables:
```

Comment:
To avoid duplication, I would simply link to `#option-2-read-credentials-from-environment-variables` (section above).